src/widget.js

```diff
@@ -86,6 +86,11 @@ export default class Widget extends Plugin {
 		const viewDocument = editor.editing.view;
 		let element = domEventData.target;
 
+		// Widget is not selectable when editor is read-only.
```

Comment:
Actually... it could. But it looks like the editor is fully functional if the image responds to clicks and hover. To whichever of these events the widget will respond it will look bad. So I'm not sure what should happen.

OTOH, if we'll keep blocking this mousedown listener then we'll allow making strange selections inside widgets and that should not be possible as well.

So – I'd do nothing now. Let's revert this change and only block the backspace/delete keys which should do nothing at all.